codecov.yml

```
@@ -32,6 +32,9 @@ ignore:
   - 'packages/@o3r-training/showcase-sdk/src/spec/**/*'
   - 'tools/github-actions/*/packaged-action/**/*'
 
+  # Templates
+  - '**/*.template'
```

Shouldn't we ignore all files in templates folder 
```suggestion
  - '**/schematics/**/templates/**/*'
```